specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -138,6 +140,16 @@ supports customers that want to:
 1. Use the shared component libraries in their projects.
 2. Build their own docker image for data-pipelines
 
+#### Docker
+A Docker image will be published on docker hub under the user https://hub.docker.com/u/versatiledatakit.
```

Comment:
nit: may rephrase the statement, so we don't need to update it upon making this change -> or alternatively have the ticket linked, so we don't forget to update it?